draft-ietf-tls-tls13.md

```diff
@@ -1355,8 +1355,9 @@ MUST return a "handshake_failure" alert and close the connection.
 When this message will be sent:
 
 > When a client first connects to a server, it is REQUIRED to send the
-ClientHello as its first message. The client will also send a
-ClientHello when the server has responded to its ClientHello with a
+ClientHello as its first message. Server which receives a different first
+message MUST send a fatal "unexpected_message" alert. The client will also send
```

Comment:
Neither the server nor the client is required to send any alert in any situation. if they send an alert, they are required to send a particular one, in particular situations, but they can always opt to not send one and instead, for example, just close the connection without sending anything.